net/knot/Makefile

```diff
@@ -157,6 +157,7 @@ CONFIGURE_ARGS += 			\
 	--enable-dbus=libdbus           \
 	--enable-quic			\
 	--disable-fastparser		\
+	--disable-redis			\
```

Comment:
Guys, are you sure that this is alright? After merging, mea culpa, I look into:
https://gitlab.nic.cz/knot/knot-dns/-/blob/5deae5465bbe117928269b6147f581eb02d384b8/configure.ac#L264

It should be ``--enable-redis=no``

Let me try to compile it first.

Reply:
It should be equivalent.

Reply:
BTW you have to look at the correct branch https://gitlab.nic.cz/knot/knot-dns/-/blob/3.5/configure.ac?ref_type=heads#L264

Reply:
Unfortunately, it still does not work. It fails due to:
```
Package knot is missing dependencies for the following libraries:
libhiredis.so.1.1.0
```

Reply:
I tried both: ``--enable-redis=no``  and as well ``--disable-redis``. Still it requires libhired library, sadly. 😢

Reply:
We will try to debug it with our colleague from Turris...